src/extension.ts

```diff
@@ -69,13 +59,30 @@ async function doActivate(_operationId: string, context: ExtensionContext): Prom
             }));
         }
 
+        if (extensionApi.onDidClasspathUpdate) {
```

Comment:
Why do we change the sequence of event listeners here?

Reply:
Just would like to register the server mode change event more earlier

Reply:
get it